google/cloud/storage/internal/metadata_parser_test.cc

```diff
@@ -15,15 +15,17 @@
 #include "google/cloud/storage/internal/metadata_parser.h"
 #include <gmock/gmock.h>
 
+using google::cloud::storage::internal::MetadataParser;
+using google::cloud::storage::internal::nl::json;
+
 /// @test Verify that we parse RFC-3339 timestamps in JSON objects.
 TEST(MetadataParserTest, ParseTimestampField) {
   std::string text = R"""({
       "timeCreated": "2018-05-19T19:31:14Z",
       "updated": "2018-05-19T19:31:24Z"
 })""";
-  auto json = storage::internal::nl::json::parse(text);
-  auto actual = storage::internal::MetadataParser::ParseTimestampField(
-      json, "timeCreated");
+  auto json = json::parse(text);
```

Comment:
Having both a variable and used namespace of `json` makes for confusion (here and lots of places below).